shoreditch.php

```diff
@@ -158,4 +160,47 @@ function shoreditch_civicrm_pageRun(&$page) {
   if ($pageName == 'CRM_Contact_Page_View_Summary') {
     CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.shoreditch', 'js/contact-summary.js');
   }
+
+  _add_shoreditch_page_styles($page->urlPath);
+}
+
+/**
+ * @return bool
+ */
+function _shoreditch_is_custom_css_enabled() {
+  $cssUrl = (string) Civi::settings()->get('customCSSURL');
+  return (bool) strpos($cssUrl, 'shoreditch/css/custom-civicrm.css');
+}
+
+/**
+ * Adds css files that match the current url path.
+ *
+ * E.g. if we are on the page civicrm/contact/view
+ * This function will search for and load the following css files if they exist.
+ *   css/civicrm.css
+ *   css/civicrm/contact.css
+ *   css/civicrm/contact/view.css
+ *
+ * @param array $path
+ */
+function _add_shoreditch_page_styles($path) {
```

Comment:
Can we have `$pathPieces` instead of `$path`? The reason is that `$path` instantly associates with a string (or is it just me?)